org.jacoco.doc/docroot/doc/faq.html

```diff
@@ -23,6 +23,15 @@ <h1>FAQ</h1>
   before.
 </p>
 
+<h3>When will feature X be released?</h3>
+<p>
+  JaCoCo is maintained by volunteer as a pure side task. As we cannot
+  guarantee free capacity we do not commit to particular release dates. In the
+  <a href="changes.html">change log</a> you can see all features which are 
+  already implemented on master and will be available with the next release.
+  Typically you can expect a couple of releases every year.
```

Comment:
Minor editorial suggestions:
> JaCoCo is maintained by volunteer[s] [in their free time(?)]. [Since] we cannot
>  guarantee free capacity[,] we do not commit to particular release dates. In the
>  \<a href="changes.html">change log</a>[,] you can see all features \[that have 
>  been] implemented on master and will be available with the next release.
>  Typically[,] you can expect a couple of releases every year.

Reply:
Thanks @bjkail :thumbsup: ❤️